tools/mdm/migration/mdmproxy/mdmproxy.go

```diff
@@ -31,7 +31,21 @@ type mdmProxy struct {
 	// mutex is used to sync reads/updates to the migrateUDIDs and migratePercentage
 	mutex sync.RWMutex
 	// token is used to authenticate updates to the migrateUDIDs and migratePercentage
-	token string
+	token      string
+	debug      bool
+	logSkipped bool
+}
+
+func skipRequest(r *http.Request) bool {
+	// Throw out a bunch of common junk requests
+	return strings.Contains(r.URL.Path, ".php") ||
+		strings.Contains(r.URL.Path, ".git") ||
+		strings.Contains(r.URL.Path, ".yml") ||
+		strings.Contains(r.URL.Path, ".txt") ||
+		strings.Contains(r.URL.Path, ".py") ||
+		strings.Contains(r.URL.Path, "wp-") ||
+		strings.Contains(r.URL.Path, "private") ||
+		(r.URL.Path == "/" && r.Method == http.MethodPost)
```

Comment:
if we find out this is not enough, we could also skip requests that don't contain `Content-Type: application/x-apple-aspen-mdm-checkin` or `Content-Type: application/x-apple-aspen-mdm`